### PR TITLE
docs update post

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Docs update post
## Reference Requirements
- Task ID: #128

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by mvn-haphan-dn — COMMENTED** [review]
LGTM

File: docs/api-references/post.md [hunk 1]
```diff
@@ -179,4 +179,29 @@ Delete the post
...
+| Key | Type | Description |
+|---|---|---|
+| id | Integer | Id post |
+| body | Text | Content of post |
+
+#### Response
+```
+{
+    "body": "new content",
+    "user_id": "5",
```
Comment (at-vietpham) [inline docs/api-references/post.md]: `user_id` trả về kiểu Integer

File: docs/api-references/post.md [hunk 2]
```diff
@@ -179,4 +179,29 @@ Delete the post
...
+#### Request headers
+| Key | Value |
+|---|---|
+|Accept|application\json
+|Authorization|{token_type} {access_token}|
+
+#### Request body
+| Key | Type | Description |
+|---|---|---|
+| id | Integer | Id post |
```
Comment (at-vietpham) [inline docs/api-references/post.md]: Khắc với example response?

File: docs/api-references/post.md [hunk 3]
```diff
@@ -179,4 +179,29 @@ Delete the post
         }
     }
 }
-```
\ No newline at end of file
+```
+### `POST` Update post
```
Comment (at-vietpham) [inline docs/api-references/post.md]: Update sao lại dùng method `POST`

**Review by ghost — CHANGES_REQUESTED** [review]

File: docs/api-references/post.md [hunk 4]
```diff
@@ -179,4 +179,38 @@ Delete the post
...
+{
+     "data": {
+        "id": 14,
+        "user_id": 9,
+        "book_id": 9,
+        "post_type": 1,
+        "body": "new content",
+        "rate_point": 3.1,
+        "status": 0,
+        "deleted_at": "2018-06-06 08:45:01",
```
Comment (at-vietpham) [inline docs/api-references/post.md]: Update sao `deleted_at` lại có giá trị được?

**Review by ghost — CHANGES_REQUESTED** [review]
Nhớ Cross review trước

**Review by mvn-haphan-dn — COMMENTED** [review]
LGTM

**Comment by at-vietpham** [comment]
LGTM 🐙

**Pull request merged**